knative-operator/deploy/resources/dashboards/grafana-dash-knative-health.yaml

```diff
@@ -0,0 +1,276 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-dashboard-definition-knative-health
```

Comment:
does this file match with an upstream version?

Reply:
@matzew no this is specific to Openshift at the moment.

Reply:
Asking, b/c there is something on upstream: https://github.com/knative/eventing/tree/master/config/monitoring/metrics

Also, @skonto I have this PR now installed, where do I see the eventing dashboard? 

I do see **Knative Serving - Revision CPU and Memory Usage** in the drop-down, but nothing for eventing :thinking:

Reply:
@matzew this is coming is not part of this PR. Adding that dashboard is part of another story.